### DISCUSSION TITLE
Possible to "debounce" package updates?

### DISCUSSION THREAD

**Discussion by laurent22** [renovatebot/renovate#19984] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have my config setup to update packages every 10 days (`stabilityDays: 10`) as I hope to reduce noise.

However, recently a package (`ts-jest`) was updated yesterday (v29.0.4) and then again today (v29.0.5). I feel this is not necessary, because by the time Renovate upgraded to version 29.0.4, version 29.0.5 was already available so it could have jumped to this directly, which means there would be just one update instead of two.

So I guess what I have in mind is something like a "debounce" which, instead of picking the version that was 10 days ago, it picks whatever is the latest version in the past 10 days. In that case it would mean upgrading to 29.0.5, and skipping 29.0.4.

Is it currently possible to achieve this with Renovate?

In case it helps, those were the two pull requests:

29.0.4: https://github.com/laurent22/joplin/pull/7651

29.0.5: https://github.com/laurent22/joplin/pull/7659

### What you tried so far.

I have read through the documentation but couldn't find a parameter that would achieve this, although it's possible I'm not looking at the problem the right way.

**Comment by viceice** [comment]
currently not possible
